src/pybert/gui/view.py

```diff
@@ -122,15 +145,47 @@ def do_load_data(self, info):
         if dialog.open() == OK:
             pybert.load_results(Path(dialog.path))
 
+    def do_clear_data(self, info):
+        """If any reference or prior results has been loaded, clear it.
+
+        Args:
+            info: When an action is clicked, it passes the whole trait instance to this function.
+        """
+        # pylint: disable=no-self-use
+        pybert = info.object
+        pybert.clear_reference_from_plots()
+
+    def toggle_debug_clicked(self, info):
+        """Toggle whether debug mode is enabled or not."""
+        # pylint: disable=no-self-use
+        info.object.debug = not info.object.debug
+
+    def getting_started_clicked(self, info):
+        """Open up Pybert's wiki."""
+        # pylint: disable=no-self-use,unused-argument
+        webbrowser.open("https://github.com/capn-freako/PyBERT/wiki")
+
+    def show_about_clicked(self, info):
+        """Open a dialog and show the user the about info."""
+        # pylint: disable=no-self-use,unused-argument
+        dialog = MessageDialog(
+            title="About Pybert",
+            message=f"PyBERT v{__version__} - a serial communication link design tool, written in Python.",
+            informative=(
+                f"{__authors__}<BR>\n" f"{__date__}<BR><BR>\n\n" f"{__copy__};<BR>\n" "All rights reserved World wide."
+            ),
+            severity="information",
+        )
+        dialog.open()
+
+    def close_app(self, info):
+        """Close the app.
+
+        Can't use CloseAction until https://github.com/enthought/traitsui/issues/1442 is resolved.
+        """
+        # pylint: disable=no-self-use,unused-argument
+        sys.exit(0)
 
-# These are the "globally applicable" buttons referred to in pybert.py,
-# just above the button definitions (approx. line 580).
-run_sim = Action(name="Run", action="do_run_simulation")
-stop_sim = Action(name="Stop", action="do_stop_simulation")
```

Comment:
What is our way of aborting a simulation, now?

Reply:
We don't currently have one.
Adding to _Issues_ as a low priority future development item...